src/graphql/search/cms/index.ts

```diff
@@ -3,29 +3,41 @@ import { TYPES } from './config'
 import getFormattedResults, { QueryCmsSearchArgs } from './normalize'
 import getCmsFilters from './filters'
 import { getCmsFromElasticSearch } from '../../../es/cms'
+import { DEFAULT_LIMIT } from '../../config'
+import getPageInfo from '../../utils/getPageInfo'
 
 const cmsSearch = (type: string) => async (
   _: any,
   { q, input }: QueryCmsSearchArgs,
 ): Promise<CmsSearchResult> => {
-  const { from, limit, filters: inputFilters, sort } = input || {}
+  let { page, limit, filters: filterInput, sort } = input || {}
+
+  // Get the page and limit from the input, otherwise use the defaults
+  page = page || 1
```

Comment:
Fancy operators here too :+1:

Reply:
In this case the `||` is better, as the page must not be zero. Using the `&&` will result in users being able to navigate to page zero

Reply:
Ok, that makes sense.